packages/wallets/src/components/WalletsAddMoreCardBanner/WalletsAddMoreCardBanner.tsx

```diff
@@ -84,7 +86,7 @@ const WalletsAddMoreCardBanner: React.FC<TWalletCarouselItem> = ({
 
                     if (!currency) return;
 
-                    if (isEuRegion) {
+                    if (isEuRegion || !hasAnyActiveRealWallets) {
```

Comment:
@lubega-deriv 
https://github.com/deriv-com/deriv-app/blob/679d73abff77c91d695f1c83f079f488efbbe4b1/packages/wallets/src/helpers/urls.ts#L149C39-L149C50

this function has maltainvest hardcoded. 
this parameter needs to be updated as well

Reply:
Updated